web/client/plugins/Dashboard.jsx

```diff
@@ -120,27 +123,42 @@ const WidgetsView = compose(
  * @prop {boolean} cfg.enabled if true, render the plugin
  * @prop {number} cfg.rowHeight Rows have a static height
  * @prop {object} cfg.cols Number of columns in this layout. default { lg: 6, md: 6, sm: 4, xs: 2, xxs: 1 }
+ * @prop {object} cfg.minLayoutWidth minimum size of the layout, below this size the widgets are listed in a single column
  * for more info about rowHeight and cols, see https://github.com/STRML/react-grid-layout#grid-layout-props
  */
 class DashboardPlugin extends React.Component {
     static propTypes = {
         enabled: PropTypes.bool,
         rowHeight: PropTypes.number,
-        cols: PropTypes.object
+        cols: PropTypes.object,
+        minLayoutWidth: PropTypes.number
     };
     static defaultProps = {
         enabled: true,
-        cols: { lg: 6, md: 6, sm: 4, xs: 2, xxs: 1 }
+        minLayoutWidth: 480
     };
     render() {
-        return this.props.enabled ? (<ContainerDimensions>{({ width, height }) => <WidgetsView width={width} height={height} rowHeight={this.props.rowHeight} cols={this.props.cols} />}</ContainerDimensions>) : null;
+        return this.props.enabled
+            ? <WidgetsView
+                width={this.props.width}
+                height={this.props.height}
+                rowHeight={this.props.rowHeight}
+                cols={this.props.cols}
+                minLayoutWidth={this.props.minLayoutWidth}
+            />
+            : null;
 
     }
 }
 
 export default {
-    DashboardPlugin,
+    DashboardPlugin: withResizeDetector(DashboardPlugin),
     reducers: {
+        dashboard: dashboardReducers,
         widgets: widgetsReducers
+    },
+    epics: {
+        ...dashboardEpics,
```

Comment:
do we need all of them here?

Reply:
If you try to add only a Dashboard plugin in a application most of the interactions were missing for example loading of a dashboard (epics/dashboard) or filter of maps synch with table (epics/widgets).

Reply:
> do we need all of them here?
@MV88 Do you think this is causing some issue?

Reply:
I'll remove them from here

Reply:
i thought it was only redundant, maybe is better to have them only in the plugin?